src/components/TextField/TextField.tsx

```diff
@@ -0,0 +1,40 @@
+interface Props {
+  title: string;
+  name: string;
+  value?: string;
+  label: string;
+  placeholder: string;
+  hasError?: boolean;
+  required?: boolean;
+  onChange: (newValue: string) => void;
+}
+
+export const TextField: React.FC<Props> = ({
+  title,
+  name,
+  value = title,
```

Comment:
[CODE STYLE] - Don't use one-letter variable naming. The default value for 'value' is using 'title', which is a one-letter variable in the context of this component. Consider renaming 'title' to something more descriptive to avoid confusion and improve code readability.